modules/platforms/cpp/ignite/schema/binary_tuple_parser.cpp

```diff
@@ -251,6 +252,12 @@ big_integer binary_tuple_parser::get_number(bytes_view bytes) {
     return big_integer(bytes.data(), bytes.size());
 }
 
+big_decimal binary_tuple_parser::get_decimal(bytes_view bytes) {
+    auto scale = load_as<std::int32_t>(bytes);
```

Comment:
The scale should be provided as A method parameter. It is not stored in binary tuple. It is defined in schema.

Reply:
Fixed